src/main.rs

```diff
@@ -498,11 +498,12 @@ impl AsyncComponent for App {
         });
 
         #[cfg(any(target_os = "macos", target_os = "linux"))]
-        let tray_img =
-            Icon::from_png_bytes(TRAY_ICON).expect("Tray icon exists and is a valid PNG; qed");
+        let img = image::load_from_memory_with_format(TRAY_ICON, image::ImageFormat::Png)
+            .expect("Tray icon is a valid PNG; qed");
+        let tray_img = Icon::from_rgba(img.to_rgba8().into_vec(), img.width(), img.height())
+            .expect("Betrayer normalization tray icon; qed");
         #[cfg(target_os = "windows")]
-        let tray_img =
-            Icon::from_resource(1, None).expect("Tray icon exists and is a valid PNG; qed");
+        let tray_img = Icon::from_resource(1, None).expect("Tray icon is a valid PNG; qed");
```

Comment:
It is not a PNG icon in Windows resource, it is [ICO](https://en.wikipedia.org/wiki/ICO_(file_format))